Tasks/Common/azure-arm-rest/azure-arm-network.ts

```diff
@@ -14,7 +14,7 @@ export class NetworkManagementClient extends azureServiceClient.ServiceClient {
     constructor(credentials: msRestAzure.ApplicationTokenCredentials, subscriptionId, baseUri?: any, options?: any) {
         super(credentials, subscriptionId);
 
-        this.apiVersion = '2016-09-01';
+        this.apiVersion = '2015-06-15';
```

Comment:
Latest api version being supported by Azure Stack is 2015-06-01 that's why had to decrease API version.